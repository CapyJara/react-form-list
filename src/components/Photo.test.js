import React from 'react';
import { shallow } from 'enzyme';
import Photo from './Photo';

describe('Photo component', () => {
  it('renders a Photo', () => {
    const photo = 'https://i.pinimg.com/originals/29/cd/f5/29cdf57a4f9526cdd04eedf435712819.jpg';

    const wrapper = shallow(<Photo photo={photo} />);
    expect(wrapper).toMatchSnapshot();
  });
});
