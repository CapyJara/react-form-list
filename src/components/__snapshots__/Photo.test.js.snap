// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo component renders a Photo 1`] = `
<img
  src="https://i.pinimg.com/originals/29/cd/f5/29cdf57a4f9526cdd04eedf435712819.jpg"
/>
`;
